### DISCUSSION TITLE
Overly bright SWCF in latest tags (post cam6_3_095?)

### DISCUSSION THREAD

**Discussion by bstephens82** [NCAR/amwg_dev#225] [TF2: Tuning]
We've noticed that in the latest tags the magnitude of SWCF is too large, more like -49 W/m^2.  I am currently running a series of out-of-the-box cam_dev tests with each tag starting with cam6_3_095, to see how this problem evolved with the latest tags or if one tag in particular is mostly responsible.  So far I can say that in the 092 tag, the out-of-the-box L58 cam_dev global mean SWCF was -47.59, while in the 095 tag it was -48.63.  My tests for the other tags should complete today, hopefully, and I can include those results here.  @adamrher I think may have more to say about this problem, I believe he thought it might be related to the "emitb4turb" update?

**Comment by adamrher** [comment]
The emitb4turb update in cam6_3_099 leads to a 25% increase in dust, and probably other species as well ... so that could serve to nucleate more clouds.

Do we have a sense what direction the SWCF will go when we couple with MOM6? Make the bias worse, or better?

**Reply by JulioTBacmeister** [reply]
Hmmm I think that's a complicated question.  My anecdotal hunch is that coupling can produce positive feedbacks, i.e. if low clouds are too bright => ocean cools => even brighter clouds and vice versa.

**Comment by bstephens82** [comment]
After finally realizing I had requested 18000 tasks for these tests and waiting for a day or two in the queue wondering what was wrong, I finally changed NTASKS to a more reasonable 1800 and they finally completed.

Here's a table of the out-of-the-box SWCF global mean values for tags 092-099 (with the exception of 093 and 094).  These are all 14-month runs, L58, cam_dev.  I make no namelist changes at all except for the 58-level grid file and interpolation.
| 092| 093 | 094 | 095 | 096 | 097 | 098 | 099 | 106 |
| -- | -- | -- | -- | -- | -- | -- | -- | -- |
|  -47.59 |  -49.00 | -48.62 | -48.63 | -50.12 | -50.87 | -50.87 | -50.02 | -50.06 |

Thus assuming I did everything correctly, it seems like the 095 tag may have increased SWCF (assuming it wasn't 093 or 094), but the 096 and 097 tags took it even higher.  The 099 tag actually seems to have brought it back somewhat, counterintuitive according to what @adamrher suggested above.  Of course this is just a set of L58 cam_dev F2000climo runs---I don't know how sensitive SWCF is to the different kinds of grid and compset combinations etc.

**Comment by bstephens82** [comment]
Adding this comment to note that I've updated the table above for the 093 and 094 tags in case my edit doesn't alert interested parties.

**Reply by adamrher** [reply]
It looks like there were two ~1.5 W/m2 changes since cam6_3_092: 

- 092->093 (PR for cam6_3_093 here: https://github.com/ESCOMP/CAM/pull/727)
- 095->096 (PR for cam6_3_096 here: https://github.com/ESCOMP/CAM/pull/760)

Both of these are mam/chemistry changes. 093 is too large a PR for me to wager a guess of what's change is responsible for brightening. 096 is a very small PR, basically it changes the definition of ```cldw``` to only contain cldliq (it was previously the sum of cldliq and cldice). This is change is confined to mozart/chemistry, not ice_nuc, so I don't know how this manifests in brighter clouds.

**Reply by bstephens82** [reply]
Hmm, well I am trying an 099 run with a slightly increased value of clubb_gamma_coef, so we can see if a small adjustment there will improve the SWCF.  I haven't looked very closely at the details of the SWCF spatial change pattern but it roughly seems to me that things are brightening across the board (ie more negative SWCF) in the later tags.

**Reply by adamrher** [reply]
alrighty. just note that it's not clear whether it's better to increase gamma or decrease c8, to increase skewness (and reduce clouds). Is this the out of the box gamma, I think 0.270?

**Reply by bstephens82** [reply]
Yes just regular gamma.  I could also try one with a smaller c8.

**Reply by tilmes** [reply]
Two points here, we need to make sure the aerosols (dust and sea-salt) are getting tuned before making large cloud adjustments.
Second, the change from 95 to 96, is a bug fix. aqueous chemistry was performed on both water and ice clouds, but we have fixed it now, to only be done on liquid clouds (as the name suggests). This increases sulfate and therefore will make clouds brighter.

**Reply by adamrher** [reply]
thanks for explanation! can you elaborate on why removing ice clouds from aq chemistry increases sulfate? (try to keep ur response in this thread box pls)

**Comment by bstephens82** [comment]
Here are the diagnostics from a couple of tests I did adjusting the values of clubb_gamma_coef and clubb_c8 for the 099 tag, L58, cam_dev, F2000climo.  In the gamma case, I used a value of 0.32 (up from 0.27), and in the c8 case, I used a value of 3.6 (down from 4.2).  Both tests show a more reasonable global mean SWCF, but it looks to me like the brights are too bright and dims are too dim in both cases.

https://webext.cgd.ucar.edu/F2000climo/dflt099_c8/atm/dflt099_c8-obs.0001_1/

https://webext.cgd.ucar.edu/F2000climo/dflt099_gamma/atm/dflt099_gamma-obs.0001_1/

**Reply by adamrher** [reply]
The Sc decks are ~gone in these new runs ... can re-run with gamma = 0.29 and another with c8 = 4.0? I think we need to aim for -47 W/m2 ... any more hits the Sc decks too hard.

**Reply by tilmes** [reply]
<img width="198" alt="image" src="https://user-images.githubusercontent.com/22222754/227609257-00d3f809-24a3-48e7-ae84-88986978ccfb.png">
Looks like AODDUST and AOD are both a little low. We could reduce the dust factor somewhat.

**Reply by bstephens82** [reply]
Is the dust factor a specific namelist parameter?  I haven't adjusted that before.

**Reply by JulioTBacmeister** [reply]
I agree that we can't seem to afford to cut back the global SWCF by only reducing subtropical stratus.  We need a parameter that targets mid and high latitude low clouds.

And I'm also nervous that adding more dust will move us in the wrong direction ... That doesn't mean we shouldn't do it., but we may need to get creative about reducing SWCF away from the subtropical StCu

**Reply by bstephens82** [reply]
> The Sc decks are ~gone in these new runs ... can re-run with gamma = 0.29 and another with c8 = 4.0? I think we need to aim for -47 W/m2 ... any more hits the Sc decks too hard.

I've got both of these going, will link here when they complete.

**Reply by bstephens82** [reply]
Here are diagnostics from updated tests using the 099 tag with [gamma=0.29](https://webext.cgd.ucar.edu/F2000climo/dflt099_gamma_0.29/atm/dflt099_gamma-obs.0001_1/) and another test with [c8=4.0](https://webext.cgd.ucar.edu/F2000climo/dflt099_c8_4.0/atm/dflt099_c8-obs.0001_1/).  I think the Sc improves, but the bright areas are still too bright in general and the dimmer areas still too dim in general.  Not sure what would cause that.  If changing the dust factor might have an impact on this just let me know what parameter to set and I'll be happy to try that out.  But my thinking is we are not going to be able to simply adjust gamma or c8 and get back to a nice looking SWCF.

**Reply by adamrher** [reply]
@tilmes suggested tuning the sea-salt emissions to thin out the clouds over the Southern Ocean. 
```
<entry id="seasalt_emis_scale" type="real" category="cam_chem"
       group="aerosol_nl" valid_values="" >
Tuning for seasalt_emis
Default: set by build-namelist.
</entry>
```
This provides very little information. Which direction does this nl need to move to reduce emissions? 

[Maybe you could point your climo files to @tilmes and she can see where we are with sea-salt w.r.t obs?]

**Reply by tilmes** [reply]
could you show me the plots you are looking at? What areas are too bright?

**Reply by bstephens82** [reply]
Hi @tilmes here are the links to the latest plots we are looking at:

clubb_gamma_coef=0.29:
https://webext.cgd.ucar.edu/F2000climo/dflt099_gamma_0.29/atm/dflt099_gamma-obs.0001_1/

clubb_c8=4.0:
https://webext.cgd.ucar.edu/F2000climo/dflt099_c8_4.0/atm/dflt099_c8-obs.0001_1/

**Reply by tilmes** [reply]
What fields are you looking at? Can you point me to the exact fields?

**Reply by tilmes** [reply]
Do you have climatologies for these diagnostics somewhere?

**Reply by bstephens82** [reply]
Sorry we are looking at SWCF:
https://webext.cgd.ucar.edu/F2000climo/dflt099_gamma_0.29/atm/dflt099_gamma-obs.0001_1/set5_6/set5_ANN_SWCF_CERES-EBAF_obsc.png

https://webext.cgd.ucar.edu/F2000climo/dflt099_c8_4.0/atm/dflt099_c8-obs.0001_1/set5_6/set5_ANN_SWCF_CERES-EBAF_obsc.png

**Reply by bstephens82** [reply]
> Do you have climatologies for these diagnostics somewhere?

Sure the climatologies are located on glade at 
/glade/scratch/stepheba/diagnostics-output/atm/climo/dflt099_gamma

/glade/scratch/stepheba/diagnostics-output/atm/climo/dflt099_c8

**Reply by tilmes** [reply]
@bstephens82 could you  add AODVISdn to  your output?

**Reply by tilmes** [reply]
what variable are you using for: seasalt_emis_scale ?

**Reply by bstephens82** [reply]
> what variable are you using for: seasalt_emis_scale ?

Just the default value = 1.00

**Reply by adamrher** [reply]
@tilmes

> @bstephens82 could you add AODVISdn to your output?

I presume you mean for future runs?

> what variable are you using for: seasalt_emis_scale ?

what variable? you mean the value? It defaults to 1.0.

**Reply by tilmes** [reply]
Yes, I am using AODVISdn to compare to satellite observations.
If we want to reduce sea-salt over the Southern Ocean to make the clouds less bright, we could try to reduce this factor, just to see what happens, maybe set it to 0.8 and see if that makes a difference?

**Reply by bstephens82** [reply]
@tilmes ok thanks, I will try an 099 run with that setting.

**Comment by bstephens82** [comment]
For reference here are diagnostics for the out-of-the-box 099 tag (L58, cam_dev, F2000climo, 14-months):
https://webext.cgd.ucar.edu/F2000climo/dflt099/atm/dflt099-obs.0001_1/

**Comment by bstephens82** [comment]
Just FYI, after speaking with @JulioTBacmeister this morning I tried several tests with modified sea salt and dust parameters, and they didn't affect SWCF much.  Thought I'd post here just so people are aware in case it's interesting.  Other than the specifified parameter change, these are out-of-the-box 099, L58, cam_dev, 1-year, F2000climo runs:
| parameter change | SWCF |
| ---- | ---- |
| seasalt_emis_scale = 0.8 | -49.66 |
| seasalt_emis_scale = 1.2 | -49.97 |
| dust_emis_fact = 0.5 | -50.79 | 
| dust_emis_fact= 0.9 | -49.80 |

I thought it was interesting that both decreasing and increasing seasalt_emis_scale caused the SWCF to drop from the out-of-the-box value (-50.02 as in the table above), albeit only slightly.

**Reply by tilmes** [reply]
@bstephens82 Just to note, if you reduce the dust_emis factor you increase the emissions. If you want to reduce dust emissions, you have to increase the factor. 
Further, when you compare sesalt_emis 0.8 and 1.2, do you see any changes in the SWCF over the Southern Ocean?

**Reply by bstephens82** [reply]
> @bstephens82 Just to note, if you reduce the dust_emis factor you increase the emissions. If you want to reduce dust emissions, you have to increase the factor. Further, when you compare sesalt_emis 0.8 and 1.2, do you see any changes in the SWCF over the Southern Ocean?

Okay thanks for that info, Julio asked me to try both increasing and decreasing the parameters, just to see.

Here are the SWCF plots for the seasalt tests:

seasalt_emis_scale = 0.8:
![set5_ANN_SWCF_CERES-EBAF_obsc](https://user-images.githubusercontent.com/59519056/228083798-33e6a674-886e-4527-bb75-46c97c762375.png)

seasalt_emis_scale = 1.2:
![set5_ANN_SWCF_CERES-EBAF_obsc](https://user-images.githubusercontent.com/59519056/228083821-60e1fbfc-de03-45cf-858a-c71fc4862e0e.png)

**Reply by tilmes** [reply]
So there is a small change with the sea salt, but not much..

**Reply by adamrher** [reply]
It would be useful to see the spatial differences between 93-92 and 96-95, in order to illustrate how chemistry and sulfates can impact the clouds (since we see so little sensitivity to dust and sea salt here).

**Reply by bstephens82** [reply]
Difference between 096-095:
https://webext.cgd.ucar.edu/F2000climo/dflt096/atm/dflt096.0001_1-dflt095.0001_1/

Difference between 093-092:
https://webext.cgd.ucar.edu/F2000climo/dflt093/atm/dflt093.0001_1-dflt092.0001_1/

**Comment by JulioTBacmeister** [comment]
Looks to me like 096-095 had more effect on the subtropical stratus than 092-093. It would be good to get a similar comparison of the sea-salt experiments even though the global effect looks tiny.  It seems to me that we should be looking at how to dim the mixed phase clouds in the Southern Ocean - more ice?

**Reply by bstephens82** [reply]
How can we get more ice?  I am looking through microphys parameters...micro_mg_dcs for example is the autoconversion size threshold, but I am assuming that's from vapor to liquid?

**Reply by adamrher** [reply]
I believe micro_mg_dcs is how @andrewgettelman got much higher IWP's in one of his runs he presented a few Friday's back. This parameter controls the threshold radius of cldice for autoconversion to snow. So a larger value will require the ice crystals to grow larger before autoconverting; the net results is more cldice. I think it currently defaults to 500E-6 ... so we could double it to 1000E-6, but Im not sure we need to kick it that hard.

**Reply by bstephens82** [reply]
Okay I will try an 099 run with a larger value of dcs if that sounds good.

**Reply by bstephens82** [reply]
It seems like a **lower** value of micro_mg_dcs improves things.  If I increase it, in this case to 1000E-6, then we get a much brighter SWCF:
![set5_ANN_SWCF_CERES-EBAF_obsc](https://user-images.githubusercontent.com/59519056/229234642-3c035964-73fa-4e83-83b4-dc6818759ad5.png)

However if I decrease it, in this case to 250E-6, then we get something more reasonable (although it still looks like the southern storm tracks are too bright, and maybe as @vlarson was suggesting this is related to dimmer subtropics in the Pacific?)
![set5_ANN_SWCF_CERES-EBAF_obsc](https://user-images.githubusercontent.com/59519056/229234520-2b8f97fe-9a51-4bd8-ac6b-6ea19c12c310.png)

**Reply by adamrher** [reply]
well, that gives me pause.

**Reply by andrewgettelman** [reply]
I've seen that behavior in DCS before (when we were tuning up CAM6).  By the way, are you running with -phys cam_dev?

**Reply by bstephens82** [reply]
> I've seen that behavior in DCS before (when we were tuning up CAM6).  By the way, are you running with -phys cam_dev?

Yes running cam_dev and 58 levels.

**Reply by bstephens82** [reply]
@andrewgettelman does 250E-6 seem like a reasonable value for micro_mg_dcs?  I guess I'm wondering whether decreasing that parameter could be part of making the SWCF look better, or what we should take away from this...

**Reply by JulioTBacmeister** [reply]
Also for @andrewgettelman:  Is there someway to target the ice/liquid partitioning of low clouds over the Souther Ocean?

**Reply by adamrher** [reply]
I think Vince had a good suggestion to vary the T threshold for partitioning to detrained ice from ZM. Here are the current defaults:

```
<clubb_detice_rad                                 > 25.0D-6             </clubb_detice_rad>
<clubb_detice_rad phys="cam_dev"                  > 61.0D-6             </clubb_detice_rad>
<clubb_detliq_rad                                 > 8.0D-6              </clubb_detliq_rad>
<clubb_detphase_lowtemp                           > 238.15D0            </clubb_detphase_lowtemp>
```
where the _rad's are the radius of detrained cldliq/cldice. We could raise the T for converting to ice, to facilitate more ice over the Southern Ocean?

**Reply by bstephens82** [reply]
@adamrher you're saying we could increase clubb_detphase_lowtemp?  If so do you have a suggestion for a value?

**Reply by adamrher** [reply]
Absent @andrewgettelman chiming in, two runs, +/-5K?

It might be useful to reference this E3SM paper looking at some of these tuning knobs (see their Appendix):

https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2018MS001350

**Reply by andrewgettelman** [reply]
Sorry, tied up in meetings all AM... some replies to @adamrher @JulioTBacmeister and @bstephens82 

1. Yes, anything between 100 e-6 and 1000 e-6 for DCS is probably 'okay' for tuning purposes. I think there is an argument that smaller ice is a bit brighter (lower DCS), or more ice is brighter (larger DCS) which is why it might go either way

2. For the liquid/ice partitioning: make sure micro_mg_berg_eff is set to 1 will give more ice, reducing the efficiency will give more liquid. I'm not sure what clubb_detphase_lowtemp but if that is the temperature at which we get ice and everything above it is liquid (which seems low, so not sure that is the case) then raising that temperature should yield more ice? 
238K is the temperature where everything should be ice (pure homogenous freezing). Try raising this 5K or 10K and see what happens....

4. I tested detrain_ice_rad and lowering clubb_detice_rad (smaller size of detrained ice) should increase the SWCF as well. We had it at 1 micron before in CAM6. Maybe lower from 61e-6 to just 5e-6 or 10 e-6?

**Reply by bstephens82** [reply]
I tried increasing the value of clubb_detphase_lowtemp by 5K and by 10K, but neither has had a large impact on SWCF.  Increasing it by 5K actually brought the mean SWCF down slightly more than increasing by 10K, maybe that's just variability in the spatial pattern.

Increasing clubb_detphase_lowtemp by 5K:
![set5_ANN_SWCF_CERES-EBAF_obsc](https://user-images.githubusercontent.com/59519056/229840826-6a079f2b-493a-4465-a696-6a9b632ddad6.png)

Increasing clubb_detphase_lowtemp by 10K:
![set5_ANN_SWCF_CERES-EBAF_obsc](https://user-images.githubusercontent.com/59519056/229840740-cf146861-d104-4741-bc38-4b50ec6c3e03.png)

**Comment by bstephens82** [comment]
Since [increasing dust_emis_fact](https://github.com/NCAR/amwg_dev/discussions/225#discussioncomment-5446209) brought the SWCF down a bit, I tried a much larger increase, dust_emis_fact=3.6.  This seems like a pretty weak tuning knob; this increase from 0.9 did bring the SWCF down another 1W/m2, but overall looks pretty similar to me in the spatial pattern.  I don't know what realistic bounds there would be on this parameter but it seems like we would have to make it quite large relative to its default value to bring the SWCF down to around -45W/m2 (near the new CERES-EBAF global mean).

dust_emis_fact=0.9
![set5_ANN_SWCF_CERES-EBAF_obsc](https://user-images.githubusercontent.com/59519056/229145891-e6532cfa-cab4-47ad-bf8b-7d695af3e8af.png)

dust_emis_fact=3.6
![set5_ANN_SWCF_CERES-EBAF_obsc](https://user-images.githubusercontent.com/59519056/229145558-0c905794-28ec-4131-a06e-1059a5db701f.png)

**Comment by tilmes** [comment]
Hi all, I checked, the most reasonable value for dust is 0.8, so if you increase the factor (decrease dust) we are getting out of reasonable tuning values. If it has little impact, I am not sure it is reasonable to do this.

**Reply by bstephens82** [reply]
Okay thanks Simone for that input.  I'm not surprised that much larger values are not realistic.

**Reply by tilmes** [reply]
note, we are going to soon have a completely new dust emission scheme, which may change dust over remote regions

**Comment by bstephens82** [comment]
We're exploring the possibility of reducing micro_mg_dcs to get the SWCF back down to a reasonable level, but this brings down the LWCF significantly.  After doing some tests I've found there is another micro parameter that will bring down the SWCF but does not affect LWCF much, micro_mg_autocon_fact.  This parameter scales autoconversion from cloud liquid to water (I don't think it affects ice, but not sure) and seems to affect low clouds more.  Here is a table of some global mean values from some tests I did with this parameter, which is 0.01 by default.  
 
![Screenshot (61)](https://user-images.githubusercontent.com/59519056/231841566-cba9275b-7d1f-4335-966c-8dc185937252.png)

As you can see the SWCF comes down steadily as this parameter is increased, but LWCF stays pretty stable.  However, we may need to bright the marine Sc off NA and SA in some way since increasing autocon_fact might weaken those clouds a bit.  Here is the SWCF from the autocon_fact=0.05 test:
![unnamed](https://user-images.githubusercontent.com/59519056/231841725-cc7e209e-96c7-4e2f-9e05-d5d2a2047e26.png)

**Reply by adamrher** [reply]
I'm wondering if we can increase the ice in high clouds (and hence, LWCF) through the T dependent threshold for partitioning ZM detrainment to liq and ice. Have you tried to increase it, so that ice forms at warmer temperatures? This is a relevant comment from andrew regarding this parameter ```clubb_detphase_lowtemp ```

> For the liquid/ice partitioning: make sure micro_mg_berg_eff is set to 1 will give more ice, reducing the efficiency will give more liquid. I'm not sure what clubb_detphase_lowtemp but if that is the temperature at which we get ice and everything above it is liquid (which seems low, so not sure that is the case) then raising that temperature should yield more ice?
238K is the temperature where everything should be ice (pure homogenous freezing). Try raising this 5K or 10K and see what happens....

 I'm thinking of a run where we keep dcs at 250, but raise this value by 10K

[edit - regarding Andrew's comment on ```micro_mg_berg_eff_factor``` this already defaults to 1 and so should be 1 in your simulations.]

**Reply by bstephens82** [reply]
I did do some tests with a larger value of [clubb_detphase_lowtemp](https://github.com/NCAR/amwg_dev/discussions/225#discussioncomment-5522593), plus 5K and plus 10K, but they did not seem to have a large (or consistent) impact on cloud forcing.  This was true for both SWCF and LWCF (global mean LWCFs were 24.16 W/m2 (+5K) and 24.24 W/m2 (+10K)).  Do you think that with a lower value of micro_mg_dcs, clubb_detphase_lowtemp would have more of an impact?  Happy to try it, just let me know.

**Reply by adamrher** [reply]
Thanks, Ben. I forgot you already pursued the clubb_detphase_lowtemp angle.

I'm concerned about the use of micro_mg_autocon_fact as it degrades the marine Sc decks. And so I'm thinking about how we can potentially recover a reasonable LWCF with dcs=250E-6. I like what dcs has done in terms of reducing the cloud forcing over the storm tracks, while not mucking up the subtropical Sc decks.

**Reply by bstephens82** [reply]
Yeah I agree, I am currently doing some tests modifying both autocon_fact and dcs, to see if there's some happy medium that preserves Sc but doesn't degrade LWCF as badly.  Let me know if you think of other parameters that might be worth testing.

**Comment by bstephens82** [comment]
Here are some pretty decent looking results from a run using micro_mg_vtrmi_factor=0.1 and micro_mg_dcs=450e-6.  I'm not sure how realistic 0.1 is for the fall velocity factor, but 450e-6 is within the reasonable range.  I haven't looked over all the other diagnostics yet.  RESTOM is 6.0+ so maybe that's bad?  
![set5_ANN_SWCF_CERES-EBAF_obsc](https://user-images.githubusercontent.com/59519056/232247894-0340bf89-cefb-44e8-8aa9-9f6bf49c617b.png)
![set5_ANN_LWCF_CERES-EBAF_obsc](https://user-images.githubusercontent.com/59519056/232247898-cc2c3930-419d-4936-b5bd-5276ded231db.png)

**Reply by bstephens82** [reply]
@andrewgettelman thinks that micro_mg_vtrmi_factor=0.1 is too extreme.  He thinks a factor of 2 (so within the range 0.5-2.0) would be more justifiable.  I will try working with micro_mg_vtrmi_factor=0.5 and tweaking some other parameters to see if I can achieve a similar result.

**Comment by tilmes** [comment]
@andrewgettelman what do you think about changing the dust scaling factor of het freez?

**Comment by adamrher** [comment]
@tilmes Ill wait for andrew to chime in, but that could be helpful. I think this parameter tends to increase ice number in NH quite a bit (b/c of all the dust sources) and it looks like from Ben's LWCF plot, that we are biased low in the NH storm track regions by about 10 W/m2. But I also recall that large changes in ice number don't always map onto changes in cldice, and so this isn't the most direct way to impact LWCF.

I dug up an email exchange with Chuck, Doug and Rolando from Mar 2022, where we had looked into modifying the ice fall velocities to address **the UTLS drying due to increased vertical resolution**, and It was effective in that regard. So It seems we've come full circle in that our UTLS moisture bias and this LWCF bias are mutually benefitting from modifying ```micro_mg_vtrmi_factor``` and ```micro_mg_dcs``` to promote more ice.

At that time, we hard coded scaling the cldice fallspeeds by 0.2. Similar to what Andrew is saying about a value of 0.1, at the time we couldn't defend the 0.2, and decided that some combination of smaller perturbation than 0.2 (like the 0.5 that Andrew is suggesting is defensible) and reducing dcs could achieve our goals. Sound familiar?

I did look into what we get out of decreasing vrtmi_factor=1.0->0.5 w.r.t the UTLS moisture bias. It's about 20-30% increase in Q:

![temp_dzonal_Q 1 0 v 0 5](https://user-images.githubusercontent.com/29961476/232532574-a022e3f2-95e4-4fbe-869b-3672b5ee79a2.png)

But compare this to changing the vtrmi_factor = 0.5->0.1, which has a much larger impact on the UTLS:

![temp_dzonal_Q 0 5 v 0 1](https://user-images.githubusercontent.com/29961476/232533098-7c919a96-8842-422e-aa91-13968dcf56f8.png)

If the Q changes are indicative of the ice changes, we may not get as pretty LWCF bias plots as in the 0.1 run, but we'll see what Ben is able to do with a value of 0.5 and lower dcs.
